lib/screens/accommodation_details-screen/accommodation_details_screen.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+import '../../assets.dart';
+import '../../models/accommodation.dart';
+import '../../http.dart';
+import '../../theme.dart';
+
+class AccommodationDetailsScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: theme,
+      debugShowCheckedModeBanner: false,
+      home: Scaffold(
+        floatingActionButton: Container(
+          padding: const EdgeInsets.symmetric(horizontal: 20),
+          width: double.infinity,
+          child: ElevatedButton(
+            onPressed: () {},
+            child: Text(
+              'BOOK YOUR STAY',
+              style: textTheme.button!.copyWith(color: ThemeColors.white),
+            ),
+          ),
+        ),
+        floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
+        resizeToAvoidBottomInset: false,
+        body: SafeArea(
+          left: false,
+          right: false,
+          //TODO ?Remove future builder and receive data from home_guests_love.screen?
+          child: FutureBuilder(
+            future: http.getAllHomes(),
+            builder: (BuildContext context, AsyncSnapshot snapshot) {
+              if (snapshot.hasError) {
+                return const Center(child: Text('error'));
+              }
+              if (snapshot.connectionState == ConnectionState.waiting) {
+                return Center(
+                    child: CircularProgressIndicator(
+                  color: ThemeColors.mint500,
+                ));
+              }
+              List<Accommodation> accommodations = snapshot.data;
+              final List<Widget> starsRating = [];
+              for (int i = 0; i < 4; i++) {
```

Comment:
You can separate this to new widget. Widget that will return list of stars depending on categorization.